ci-operator/step-registry/ignition/bastionhost/ignition-bastionhost-commands.sh

```diff
@@ -90,7 +103,7 @@ acl authenticated proxy_auth REQUIRED
 acl CONNECT method CONNECT
 http_access allow authenticated
 http_port 3128
-dns_v4_first on
+${proxy_dns_config}
```

Comment:
Can proxy have such capabilities: a) ipv4 only (default) b) ipv6 only (deny ipv6) c) ipv4 first d) ipv6 first, so the proxy is more flexible, it could suite for a wide range of scenarios.

Reply:
Because this proxy is also used for e2e testing, maybe we still need to keep the ipv4 capacity.